src/chocolatey/docfx/docfx.nuspec

```diff
@@ -0,0 +1,72 @@
+    <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
+    <version>1.4</version>
+    <packageSourceUrl>https://github.com/dotnet/docfx</packageSourceUrl>
+    <!-- owners is a poor name for maintainers of the package. It sticks around by this name for compatibility reasons. It basically means you. -->
+    <owners>Microsoft</owners>
+    <!-- ============================== -->
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <!-- This section is about the software itself -->
+    <title>DocFX</title>
+    <authors>Microsoft</authors>
+    <projectUrl>http://dotnet.github.io/docfx/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/3d0ce42525e12ad5b9cd528e8cbab011d8db4737/icons/docfx.svg</iconUrl>
+    <copyright>© 2015-2016 Microsoft</copyright>
+    <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/dotnet/docfx</projectSourceUrl>
+    <docsUrl>http://dotnet.github.io/docfx/</docsUrl>
+    <!--<mailingListUrl></mailingListUrl>-->
+    <bugTrackerUrl>https://github.com/dotnet/docfx/issues</bugTrackerUrl>
+    <tags>docfx markdown documentation codecomments GithubFlavoredMarkdown GFM DFM</tags>
+    <summary>DocFX makes it extremely easy to generate your API reference, landing page, and how-to.</summary>
+    <description>DocFx is a documentation generation tool for api reference and markdown files!
+
+DocFX is an API documentation generator for .NET, currently supporting C# and VB, as similar to JSDoc or Sphinx. It has the ability to extract triple slash comments out from your source code. What's more, it has syntax to link additional files to API to add additional remarks. docfx will scan your source code and your additional conceptual files and generate a complete HTML documentation website for you. docfx provides the flexibility for you to customize the website through templates. We currently have several embedded templates, including websites containing pure static html pages and also website managed by AngularJS. Of cause, if you are interested in creating your own website with your own styles, you can follow [how to create custom template to create custom templates](http://dotnet.github.io/docfx/tutorial/howto_create_custom_template.html) to create custom templates.
+
+### Features
+
+* Generate API Documentation - Able to generate API documentation from triple-slash-comments for .NET source code directly!
+* Generate Conceptual Documentation - Able to generate HTML from markdown files supporting DFM syntax.
+* Easy Customization - Able to customize templates and themes easily
+</description>

```

Comment:
I gleaned most of the description from different places in the documentation. I did fix a bit of the wording, let me know if this is sufficient or if you want to add more.